webapp/packages/plugin-connections/src/ConnectionForm/Options/ConnectionOptionsTabService.ts

```diff
@@ -160,12 +162,14 @@ export class ConnectionOptionsTabService extends Bootstrap {
     }
 
     const driver = await this.dbDriverResource.load(state.config.driverId, ['includeProviderProperties']);
+    const driverDatabases = await this.connectionInfoResource.searchDatabases(driver.defaultDatabase || 'localhost');
```

Comment:
you can't run this function that way cuz it will:
1. make many requests to provided hosts (that may take long time)
2. will not be cached (and this is expected)

please check `select` in the webapp/packages/plugin-connection-search/src/Search/ConnectionSearchService.ts`
we use `setConfig` to provide settings from selected driver, but it seems like this settings is not honored or overwrited